pkg/tracing/config.go

```diff
@@ -96,6 +95,11 @@ func configure(serviceName string, options *Options, nz newZipkin) (io.Closer, e
 
 	reporters := make([]jaeger.Reporter, 0, 3)
 
+	sampler, err := jaeger.NewProbabilisticSampler(options.SamplingRate)
```

Comment:
I should add that we need *a* `Sampler` here.  The `ConstSampler` set to not sample didn't work in my testing and the other `Samplers` didn't seem appropriate. This gives us the default behavior we want with the ability to crank up tracing of Mixer pods in case of emergency debugging.

Reply:
sounds good.
/lgtm